src/services/connectionService.ts

```diff
@@ -142,56 +142,4 @@ const addDefaultConnections = async () => {
       )
     })
   }
-
-  const firm24Name = 'Firm24'
```

Comment:
this only removes the firm24 connection. We want to remove both connections and the call to addDefaultConnections. just remove the whole function

Reply:
Removed the whole function now.